apps/researcher/src/app/[locale]/communities/page.tsx

```diff
@@ -0,0 +1,21 @@
+import {clerkClient} from '@clerk/nextjs';
+import {getTranslations} from 'next-intl/server';
+
+// For now this is a basic page without design, just for navigating between communities.
+export default async function CommunitiesPage() {
+  const organizations = await clerkClient.organizations.getOrganizationList();
```

Comment:
Or should we also use `@/lib/community` here (rather than the Clerk client directly)?